### ISSUE TITLE
GPU support?

### ISSUE THREAD

**Issue by ahundt** [dstamoulis/single-path-nas#1]
This looks very interesting but I don't have TPU access. Do you have any plans to tweak the code to also support running on GPU/CPU?

**Comment by dstamoulis** [comment]
@ahundt indeed, great suggestion. We are investigating extensions and GPU (PyTorch) support is definitely one of them; will keep you posted!

**Comment by twmht** [comment]
@dstamoulis 

any progress on gpu support?

**Comment by fabbrimatteo** [comment]
Look at [this](https://github.com/dstamoulis/single-path-nas/issues/3) issue.

**Comment by zychen2016** [comment]
> Look at [this](https://github.com/dstamoulis/single-path-nas/issues/3) issue.

Have you solved the problem about running on GPU？could you share？  @fabbrimatteo
